pkg/k8shandler/visualization.go

```diff
@@ -158,9 +205,27 @@ func (clusterRequest *ClusterLoggingRequest) removeKibana() (err error) {
 	return nil
 }
 
-func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaDeployment(proxyConfig *configv1.Proxy, trustedCABundleCM *v1.ConfigMap) (err error) {
+func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaDeployment(proxyConfig *configv1.Proxy) (err error) {
+
+	kibanaTrustBundle := &v1.ConfigMap{}
+
+	// Create cluster proxy trusted CA bundle.
+	if proxyConfig != nil {
+		err = clusterRequest.createOrUpdateTrustedCABundleConfigMap(constants.KibanaTrustedCAName)
```

Comment:
why is this being done repeatedly?  we should be able to just define this CM as a resource in the CSV and be done with it. Same for the one for fluentd.

Reply:
sorry i take that back... CVO allows create-only resources but i dunno if OLM does.  So if you can't tell OLM to just create the CM but not continue to reconcile/stomp it, then you need to create it yourself.  Still not sure you need to actively reconcile it (there should be no changes to apply)

Reply:
so to have OLM create the CMs, we would need to define them in the manifest, correct?
It may also make sense to move the creation of these configmaps into the proxy controller... I was keeping the call in this file because that's where it was before and the call itself should be idempotent.

Reply:
yeah, as i said i don't think you can move them into the manifest after all because OLM is going to keep stomping the content of the configmap after the CAs get injected, which would be bad.  CVO has the ability to create a resource for you but then never touch it again, I don't think OLM can do that.

i think you can leave this here for now.